spearhead/variables.css

```diff
@@ -80,6 +80,12 @@
 	--sticky-posts--color-background: #F9F9F9;
 	--sticky-posts--entry-title-font-size: 24px;
 	--sticky-posts--entry-content-font-size: 20px;
+
+	/* Forms */
```

Comment:
I think these should be part of Seedlet

Reply:
Those variables do come from Seedlet, I'm just overriding their values here